### PR TITLE
Update clickhouse-keeper.md

### PR DESCRIPTION
**Pull request by blanklin030**
Changelog category (leave one):

- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
@blanklin030 do you have any difficulties signing a CLA?

**Review by rfraposa — APPROVED** [review]
LGTM

File: docs/zh/operations/clickhouse-keeper.md [hunk 1]
```diff
@@ -1 +1,323 @@
-../../en/operations/clickhouse-keeper.md
\ No newline at end of file
+---
+sidebar_position: 66
+sidebar_label: ClickHouse Keeper
+---
+
+# [预发生产] ClickHouse Keeper {#clickHouse-keeper}
```
Comment (amosbird) [inline docs/zh/operations/clickhouse-keeper.md:6]: `[预发生产]` 词不达意，建议删除

File: docs/zh/operations/clickhouse-keeper.md [hunk 2]
```diff
@@ -1 +1,323 @@
...
+
+ClickHouse 服务为了 [副本](../engines/table-engines/mergetree-family/replication.md) 和 [分布式DDL](../sql-reference/distributed-ddl.md) 查询执行使用 [ZooKeeper](https://zookeeper.apache.org/) 协调系统. ClickHouse Keeper 和 ZooKeeper是相互兼容的，可互相替代.
```
Comment (amosbird) [inline docs/zh/operations/clickhouse-keeper.md:8]: 英文语序 -> 中文语序


ClickHouse 依赖  [ZooKeeper](https://zookeeper.apache.org/) 进行[副本](../engines/table-engines/mergetree-family/replication.md)协调和[分布式 DDL](../sql-reference/distributed-ddl.md) 执行。

File: docs/zh/operations/clickhouse-keeper.md [hunk 3]
```diff
@@ -1 +1,323 @@
...
+
+!!! warning "警告"
+    这个功能当前还在预发生产阶段. 我们只是在内部部分使用于生产环境和测试CI中.
```
Comment (amosbird) [inline docs/zh/operations/clickhouse-keeper.md:11]: 机翻错误

这个功能当前还未达到生产可用阶段，仅在我们内部的生产环境与测试 CI 中完成了验证。

File: docs/zh/operations/clickhouse-keeper.md [hunk 4]
```diff
@@ -1 +1,323 @@
...
+
+## 实现细节 {#implementation-details}
+
+ZooKeeper最早的非常著名的开源协调系统之一. 它是通过Java语言实现的, 有一个相当节点和强大的数据模型. ZooKeeper的协调算法叫做 ZAB (ZooKeeper Atomic Broadcast) zk不能保证读取的线性化, 以为每个zk节点服务都是通过本地线性读的. ClickHouse Keeper是通过C++写的，和zookeeper不一样， ClickHouse Keeper使用的[RAFT algorithm](https://raft.github.io/) [implementation](https://github.com/eBay/NuRaft)算法. 这个算法允许线性读和写, 已经有几种不同的语言的开源实现.
```
Comment (amosbird) [inline docs/zh/operations/clickhouse-keeper.md:15]: 格式未统一：

1. 中文还是英文标点？
2. 中英间距有无空格？
3. zk 缩写是否合理？

File: docs/zh/operations/clickhouse-keeper.md (hunk 1 above)
Comment (amosbird) [reply docs/zh/operations/clickhouse-keeper.md:6]: 英文文档已标识生产可用，需要更新

**Comment by rfraposa** [comment]
This PR is not going to work as-is. The file being edited is a symlink, which can not be edited. The symlink has to be deleted and a new .md file added. I tested it here https://github.com/ClickHouse/ClickHouse/pull/36598 and it appears to work.

@blanklin030  - can you fix this PR so it looks like https://github.com/ClickHouse/ClickHouse/pull/36598? Thanks!

**Review by amosbird — COMMENTED** [review]
The doc contains machine translated materials, which is understandable but not friendly to reader. I think it's ok to merge in general. We can improve in future iterations.

**Pull request closed without merging**